djiffy/importer.py

```diff
@@ -193,6 +194,12 @@ def import_manifest(self, manifest, path):
             # check if this page is the thumbnail image
             if thumbnail_id is not None and db_canvas.iiif_image_id == thumbnail_id:
                 db_canvas.thumbnail = True
+
+            # check for any extra_data, right now only rendering
+            db_canvas.extra_data = OrderedDict()
```

Comment:
Can we set the field default to `OrderedDict` instead of `dict` so you can skip this step?

(I realize you're probably basing this on code that I wrote, but if you agree let's fix that too.)

Reply:
Sure. This sounds good to me. I've done just that.